webextension/background.js

```diff
@@ -862,7 +947,7 @@ async function handleButtonClick(command, tabState) {
   switch (tabState.slide) {
     case "initialPrompt": {
       const siteWorks = command === "yes";
-      pingTelemetry({satisfiedSitePrompt: yesOrNo(siteWorks)});
+      Config.maybeSendTelemetry({satisfiedSitePrompt: yesOrNo(siteWorks)});
```

Comment:
I don't know where yes or no is.   For ease of casting back at parquet (which is SQL ish), it's totally okay to send true false as strings.    I think parquet-ish sql is can do cast(field as boolean) even from "true|false".

Reply:
In this case I was just following our own spec (#2) which uses `yes` or `no`. That method is literally just converting anything truthy into the string `yes` or otherwise returning `no`; nothing terribly interesting.